testdata/README.md

```diff
@@ -0,0 +1,21 @@
+# Test data
+
+The JSON files within this directory hiearchy represent test data
+for the schema in the repository. The directory structure matches
+that in the [proto](../proto) directory, namely the protobuf package
+hierarchy.
+
+Each JSON file should be named according to the data message it
+represents, with an optional description suffix. Examples:
+
+- `StorageObjectData.json`
+- `DocumentEventData-simple.json`
+- `DocumentEventData-complex.json`
+
+When the proto schema is updated in a non-trivial way, either a new
+file should be created or an existing file should be modified, to
```

Comment:
A new _json_ file?

Reply:
I think that's reasonably implicit here, but it does no harm. Added.